MapboxCoreNavigation/String.swift

```diff
@@ -0,0 +1,24 @@
+#if os(iOS) || os(tvOS)
+    import UIKit
+#elseif os(watchOS)
+    import WatchKit
+#endif
+
+extension String {
+    static var systemName: String {
```

Comment:
These properties shouldn’t extend String just because they’re of type String. ProcessInfo would be a better type to extend.